SConstruct

```diff
@@ -159,6 +159,61 @@ logger.info(
     f"SCons {SCons.__version__} is using the following Python interpreter:\n"
     f"    {sys.executable} (Python {python_version})", print_level=False)
 
+cantera_version = "3.1.0a2"
+# For use where pre-release tags are not permitted (MSI, sonames)
+cantera_pure_version = re.match(r'(\d+\.\d+\.\d+)', cantera_version).group(0)
+cantera_short_version = re.match(r'(\d+\.\d+)', cantera_version).group(0)
+
+try:
+    cantera_git_commit = get_command_output("git", "rev-parse", "--short", "HEAD")
+    logger.info(f"Building Cantera from git commit {cantera_git_commit!r}")
+except (subprocess.CalledProcessError, FileNotFoundError):
+    env["git_commit"] = "unknown"
+
+# Python Package Settings
+python_min_version = parse_version("3.8")
+# Newest Python version not supported/tested by Cantera
+python_max_p1_version = parse_version("3.13")
+# The string is used to set python_requires in setup.cfg.in
+py_requires_ver_str = f">={python_min_version}"
+
+if "sdist" in COMMAND_LINE_TARGETS:
+    if "clean" in COMMAND_LINE_TARGETS:
+        COMMAND_LINE_TARGETS.remove("clean")
```

Comment:
I actually really liked running `scons clean sdist` so I knew it was a clean directory. The clean happens before the sdist here, since the clean also happens at "configure" time. I'd rather keep it